src/main/java/org/example/racinggame/controller/RacingCarController.java

```diff
@@ -0,0 +1,42 @@
+package org.example.racinggame.controller;
+
+import org.example.racinggame.model.Cars;
+import org.example.racinggame.view.InputView;
+import org.example.racinggame.view.OutputView;
+
+public class RacingCarController {
+
+    private final InputView inputView;
+    private final OutputView outputView;
+
+    public RacingCarController(InputView inputView, OutputView outputView) {
+        this.inputView = inputView;
+        this.outputView = outputView;
+    }
+
+    public void play() {
+        String carNamesInputString = inputView.getCarNamesFromUserInput();
```

Comment:
이 변수가 입력받은 값이라는 것을 굳이 알 필요가 있을까요? 🤔
`carNames`와 같은 네이밍이 더 적절할 것 같아요. 

```suggestion
        String carNames = inputView.getCarNamesFromUserInput();
```